backend/UserManagement.go

```diff
@@ -91,17 +92,18 @@ type LiProfile struct {
 			Title   string `json:"title"`
 		} `json:"values"`
 	} `json:"positions"`
-	Summary string `json:"summary"`
+	Summary       string `json:"summary"`
+	ShareLocation bool   `json:"shareLocation"`
+	Greeting      string `json:"greeting"`
 }
 
 // User is user on MeetOver
 type User struct {
-	ID           string         `json:"uid,omitempty"`
-	Location     *Geolocation   `json:"location,omitempty"`
-	AccessToken  ATokenResponse `json:"accessToken"`
-	LiProfile    LiProfile      `json:"li_profile"`
-	IsSearching  bool           `json:"profile"`
-	HelloMessage string         `json:"hello_message"` // TODO: ask user to fill this feild in account creation
```

Comment:
are we not having a greeting at the user profile? That is what hello_message was

Reply:
It got moved to the profile object.